core/scripts/local/CalculateContractBytecode.js

```diff
@@ -0,0 +1,24 @@
+// This simple script tells you how big your contract byte code is and how much you have until you exceed
+// the current block limit as defined by EIP170.
+
+const argv = require("minimist")(process.argv.slice(), { string: ["contract"] });
+const contractName = argv.contract;
+
+if (!contractName) {
+  console.log("Please enter the contract name as a parameter as `--contract <name>`.");
+  return;
+}
+var child = require("child_process").exec("truffle compile");
+child.stdout.pipe(process.stdout);
+child.on("exit", function() {
+  console.log("finished compiling 🚀!");
+  console.log("loading", contractName + ".json");
+  let obj = require("./../../build/contracts/" + contractName + ".json");
```

Comment:
Is this script assuming the user is running it from a particular directory? If so, what directory? Generally, we assume the user is running from either the top level or the `core` directory.

Reply:
this assumes you are running it from the core directory as with the other truffle related commands. I added a comment at the top of the file to explain this. If it cant find the file it ends relatively gracefully and says it cant find the path to the .json file.

Reply:
Oh, I see, this path is relative to the *file* because it's a require. Ignore my comment. This makes sense because the relative positions should always be the same :)